addons/spectator/functions/fn_drawTags.sqf

```diff
@@ -30,7 +30,7 @@ private _renderGroups = _grpTagSize > 0;
     };
 
     // check if the average pos is on the screen
-    private _render = [_avgpos] call FUNC(onScreen);
+    private _render = [_avgpos] call FUNC(onScreen) && !_isAI;
```

Comment:
Suggest lazy eval to avoid the function call if AI - `private _render = !_isAI && {[_avgpos] call FUNC(onScreen)}`